cpp/src/parquet/arrow/arrow_schema_test.cc

```diff
@@ -139,6 +139,10 @@ TEST_F(TestConvertParquetSchema, ParquetFlatPrimitives) {
   arrow_fields.push_back(
       ::arrow::field("time64", ::arrow::time64(TimeUnit::MICRO), false));
 
+  parquet_fields.push_back(
+      PrimitiveNode::Make("duration[us]", Repetition::REQUIRED, ParquetType::INT64));
+  arrow_fields.push_back(::arrow::field("duration[us]", INT64, false));
```

Comment:
What is the purpose of this? It's just an INT64 field like any other, no?

Reply:
Ah, I initially wrote the test with duration type, but because this test is for the pure Parquet->Arrow schema conversion (without any usage of ARROW:schema), it failed and I changed it to int64. Which of course renders it useless as it's indeed just testing int64 type. Will remove.